.gitignore

```diff
@@ -1,3 +1,4 @@
 coverage/
 node_modules/
 js/
+.vscode/
```

Comment:
This should go in your [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).